aspnet/docfx.json

```diff
@@ -36,9 +36,10 @@
     "overwrite": [],
     "externalReference": [],
     "globalMetadata": {
-      "_navPath": "/foo",
+            "_navPath": "/foo",
       "_navRel": "/foo",
       "breadcrumb_path": "/aspnet/breadcrumb/toc.json",
+      "recommendations": true,
```

Comment:
This is the only change from the pre- recommendations version. All the rest is white space the editor might have added.